docs/integrations/cicd/octopus-deploy.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Octopus Deploy"
+description: "Learn how to sync secrets from Infisical to Octopus Deploy"
+---
+
+Prerequisites:
+
+- Set up and add secrets to [Infisical Cloud](https://app.infisical.com)
+
+<Steps>
+    <Step title="Create a Service Account for Infisical in Octopus Deploy">
+        Navigate to **Configuration** > **Users** and click on the **Create Service Account** button.
+
+        ![integrations octopus deploy
+        users](/images/integrations/octopus-deploy/integrations-octopus-deploy-user-settings.png)
+
+        Fill out the required fields and click on the **Save** button.
+        ![integrations octopus deploy service
+        account](/images/integrations/octopus-deploy/integrations-octopus-deploy-create-service-account.png)
+    </Step>
+    <Step title="Generate an API Key for your Service Account">
+        On the **Service Account** user page, expand the **API Keys** section and click on the **New API Key** button.
+
+        ![integrations octopus deploy
+        new api key](/images/integrations/octopus-deploy/integrations-octopus-deploy-create-api-key.png)
+
+        Fill out the required fields and click on the **Generate New** button.
+
+        ![integrations octopus deploy
+        generate api key](/images/integrations/octopus-deploy/integrations-octopus-deploy-generate-api-key.png)
+
+        Copy the generated **API Key** and click on the **Close** button.
+
+        ![integrations octopus deploy
+        copy api key](/images/integrations/octopus-deploy/integrations-octopus-deploy-copy-api-key.png)
+    </Step>
+    <Step title="Create a Service Accounts Team and assign your Service Account">
+        Navigate to **Configuration** > **Teams** and click on the **Add Team** button.
+
+        ![integrations octopus deploy
+        teams](/images/integrations/octopus-deploy/integrations-octopus-deploy-team-settings.png)
+
+        Create a new team for **Service Accounts** and click on the **Save** button.
+        ![integrations octopus deploy add
+        team](/images/integrations/octopus-deploy/integrations-octopus-deploy-create-team.png)
+
+        On the **Members** tab, click on the **Add Member** button, add your **Infisical Service Account** and click on the **Add** button.
+        ![integrations octopus deploy add service account to team](/images/integrations/octopus-deploy/integrations-octopus-deploy-add-to-team.png)
+
+        On the **User Roles** tab, click on the **Include User Role** button, add the **Project Contributor** role and click on the **Apply** button.
+        ![integrations octopus deploy add user roles to team](/images/integrations/octopus-deploy/integrations-octopus-deploy-add-role.png)
```

Comment:
I think it would be better if we covered this "scope" configuration as wel where they can access of a service account to a specific env 
<img width="750" alt="image" src="https://github.com/user-attachments/assets/6091bcb3-05c9-4fa9-a812-d3e93f270abb">

Reply:
will do!

Reply:
I added context about this and updated the image but don't feel we need to show the modal as it's optional

![CleanShot 2024-11-25 at 13 14 03@2x](https://github.com/user-attachments/assets/0d03dd51-b508-43ba-84ff-ae5fe4b97eea)